students/younggeon-yang/user/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from user.views  import UserView, LoginView
+
+urlpatterns = [
+    path('/', UserView.as_view()),
```

Comment:
```suggestion
    path('/signup', UserView.as_view()),
```
위와 같이 하는 것이 조금 더 명확하게 회원가입을 나타내지 않을까요~~?